checkstyle.xml

```diff
@@ -207,7 +207,7 @@
         <module name="AvoidStarImport"/>
         <!-- ban new references to old commons lang and configuration -->
         <module name="IllegalImport">
-            <property name="illegalPkgs" value="sun, org.apache.commons.lang, org.apache.commons.configuration"/>
+            <property name="illegalPkgs" value="sun, org.apache.commons.lang, org.apache.commons.configuration, org.apache.commons.httpclient"/>
```

Comment:
Just curious, how did this come up? Were we using two http client libraries?

Reply:
Went through this kind of thing a while back. 
https://debuguide.blogspot.com/2013/01/quick-guide-for-migration-of-commons.html